test/components/async-route.spec.js

```diff
@@ -75,21 +80,33 @@ describe('AsyncRoute', () => {
   describe('after a load error', () => {
     it('shows a danger alert', async () => {
       setLoader('MyComponent', () => Promise.reject());
-      const asyncRoute = mountComponent({ componentName: 'MyComponent' });
+      const reload = sinon.fake();
+      const asyncRoute = mountComponent({
+        props: { componentName: 'MyComponent' },
+        container: {
+          location: { reload }
+        }
+      });
       await wait();
       asyncRoute.should.alert('danger');
+      asyncRoute.vm.$container.alert.ctaHandler();
```

Comment:
Here we call `alert.ctaHandler()` directly. We can't click `.alert-cta` because the alert isn't rendered in the DOM. This test just mounts the `AsyncRoute` component, not the whole. We independently test `alert.ctaHandler()` and `.alert-cta` elsewhere, so I think it's OK to call `alert.ctaHandler()` directly here.